demos/common/cpp/models/src/image_model.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+// Copyright (C) 2018-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#include "models/detection_model.h"
+#include <utils/ocv_common.hpp>
+#include <utils/slog.hpp>
+
+using namespace InferenceEngine;
```

Comment:
You doesnt use it here. It could be removed

Reply:
Done